main.go

```diff
@@ -24,6 +25,11 @@ var (
 
 	frmt         = app.Command("format", "Format Gunk code.")
 	frmtPatterns = frmt.Arg("patterns", "patterns of Gunk packages").Strings()
+
+	dmp         = app.Command("dump", "Write a FileDescriptorSet (a protocol buffer, defined in descriptor.proto)")
+	dmpPatterns = dmp.Arg("patterns", "patterns of Gunk packages").Strings()
+	dmpFile     = dmp.Flag("file", "file to write FileDescriptorSet").String()
+	dmpFormat   = dmp.Flag("format", "output format to write FileDescriptorSet as (options are 'raw' or 'json'").String()
```

Comment:
I was mimicking `kubectl get` command for this, and it is also trivial to output as yaml (which I find easier to debug with). I want to make this output yaml as well, so it might be better to stick with having it as a string flag, rather than a boolean flag.

Reply:
Ah, gotcha - then SGTM